README.md

```diff
@@ -2,20 +2,118 @@
 
 > Test client for Xo-Server
 
-## Usage
-
-xo-server needs to be installed on `localhost:9000`.
-You should also have a `admin@admin.net` account with password `admin`.
+## Adding a test
 
-There should be a Xen Server available at `192.168.1.3`with the credentials `root` and password `qwerty`.
+### Organization
 
-Warning ! This client will create and delete a lot of items (users, groups, servers ...) to test features
+```
+src
+├─ user
+|   ├─ __snapshots__
+|   |     └─ index.spec.js.snap 
+|   └─ index.spec.js
+├─ job
+¦   └─ index.spec.js
+¦
+¦
+├─ _xoWithTestHelpers.js
+└─ util.js
+```
 
+The tests can describe xo methods or scenarios:  
 ```
-> npm ci
-> npm test
+describe("user", () => {
+
+  // testing a method
+  describe(".set()", () => {
+    it("sets an email", async () => {
+      // some tests using xo methods and helpers from _xoWithTestHelpers.js
+      id = await xo.createUser(SIMPLE_USER);
+      expect(await xo.call("user.set", params)).toBe(true);
+      expect(omit(await xo.getUser(id), "id")).toMatchSnapshot();
```

Comment:
Use [property matchers](https://jestjs.io/docs/en/snapshot-testing#property-matchers) instead of omitting `id`.